installing/install_config/installing-customizing.adoc

```diff
@@ -22,13 +22,19 @@ when installing bare-metal nodes.
 The following sections describe features that you might want to
 configure on your nodes in this way.
 
-
+include::modules/installation-special-config-butane.adoc[leveloffset=+1]
 include::modules/installation-special-config-kargs.adoc[leveloffset=+1]
 ifdef::openshift-webscale[]
 include::modules/installation-special-config-rtkernel.adoc[leveloffset=+1]
 endif::openshift-webscale[]
 include::modules/installation-special-config-kmod.adoc[leveloffset=+1]
+
+See xref:../installing-customizing.adoc#installation-special-config-butane_installing-customizing[Configuring machine configs with Butane] for information on Butane.
```

Comment:
Should these lines be moved inside the modules, before we tell the user to run Butane, rather than afterward?  That might be better even if we can't cross-link.